### ISSUE TITLE
Binarizing a metric set

### ISSUE THREAD

**Issue by michellekarunaratne** [AstraZeneca/rexmex#46]
Hi,

is it possible binarize a metric set instead of a single metric?

**Comment by benedekrozemberczki** [comment]
We could write a utility method and create a new release. Would that help?

**Comment by michellekarunaratne** [comment]
yeah that would be much helpful. Also will it be compatible with the score card report generation? if not it wont be much useful.
